sources/engine/Stride.Assets.Models/ImportModelCommand.cs

```diff
@@ -28,10 +28,10 @@ public abstract partial class ImportModelCommand : SingleFileImportCommand
 
         public static ImportModelCommand Create(string extension)
         {
-            if (ImportFbxCommand.IsSupportingExtensions(extension))
-                return new ImportFbxCommand();
-            if (ImportAssimpCommand.IsSupportingExtensions(extension))
-                return new ImportAssimpCommand();
+           // if (ImportFbxCommand.IsSupportingExtensions(extension))
```

Comment:
Do not comment unneeded code, because it may be confusing 🙂 Please remove these lines

Reply:
Done!